### PR TITLE
feat: add vr support

### PR DESCRIPTION
**Pull request by FlayaN**
Adds supports for VR!

Requires the latest version of `VR Address Library for SKSEVR`(0.134.0) due to having to add 3 ids for this VR port.

### CODE REVIEW

File: src/Hooks.cpp [hunk 1]
```diff
@@ -356,7 +356,7 @@ namespace NLA
 		void Install() {
 			const REL::Relocation<std::uintptr_t> combatControllerUpdate{ RELOCATION_ID(43162, 44384) };
 			const REL::Relocation<std::uintptr_t> weaponFire{ RELOCATION_ID(17693, 18102) };
-			const REL::Relocation<std::uintptr_t> launchSpell{ RELOCATION_ID(33672, 34452) };
+			const REL::Relocation<std::uintptr_t> launchProjectile{ RELOCATION_ID(42928, 44108) };
```
Comment (FlayaN) [inline src/Hooks.cpp]: This change was done due to remove the offset, and also this was actually already reversed for VR, so changed this when looking for matching VR ids

Comment (adya) [reply src/Hooks.cpp]: Sorry, I didn't get that. This was supposed to hook one specific call to `LaunchProjectile` where NPCs launching a spell. But with this change is it now hooking all calls to `LaunchProjectile`?

Comment (FlayaN) [reply src/Hooks.cpp]: Oh.. ofc... haha my bad, then I guess it's 3 missing IDS for VR then :D

Comment (FlayaN) [reply src/Hooks.cpp]: I'll revert this change!

Comment (adya) [reply src/Hooks.cpp]: I see, well, I had VR exe somewhere laying around. Thanks for setting it up! Do you know how could I pass like 3 IDs together in a single relocation?

File: main.ps1 [hunk 2]
```diff
@@ -14,7 +14,7 @@
                           -CMAKE_BUILD_CONFIGURATION: Debug `
                           -FOMOD_INCLUDE_PDB $true `
                           -FOMOD_REQUIRED_INSTALLATION_DIR 'FOMOD/Required Files' `
-                          -FOMOD_MOD_NAME 'NPCs Names Distributor Debug' `
```
Comment (adya) [inline main.ps1:17]: oops, busted 😄 to be honest .ps1 file is redundant and is more for local debugging of CI script.

Comment (FlayaN) [reply main.ps1:17]: Yeah I just changed it when I saw it locally :P Was kinda nice to use it to test the fomod locally

**Pull request merged**